examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/migrate.py

```diff
@@ -20,23 +29,19 @@
 
 defs = build_defs_from_airflow_instance(
     airflow_instance=airflow_instance,
-    orchestrated_defs=defs_from_factories(
-        CSVToDuckdbDefs(
-            name="load_lakehouse__load_iris",
-            csv_path=Path("iris.csv"),
-            duckdb_path=Path(os.environ["AIRFLOW_HOME"]) / "jaffle_shop.duckdb",
-            columns=[
-                "sepal_length_cm",
-                "sepal_width_cm",
-                "petal_length_cm",
-                "petal_width_cm",
-                "species",
-            ],
+    orchestrated_defs=combine_defs(
+        defs_from_lakehouse(
+            dag_id="load_lakehouse",
+            task_id="load_iris",
+            csv_path=CSV_PATH,
+            duckdb_path=DB_PATH,
+            columns=IRIS_COLUMNS,
         ),
-        DbtProjectDefs(
-            name="dbt_dag__build_dbt_models",
-            dbt_manifest=dbt_project_path() / "target" / "manifest.json",
-            project=DbtProject(project_dir=dbt_project_path()),
+        defs_from_airflow_dbt(
+            dag_id="dbt_dag",
+            task_id="build_dbt_models",
+            manifest=dbt_manifest_path(),
+            project=DbtProject(dbt_project_path()),
         ),
```

Comment:
I think this is an incremental improvement but I would still like a format where it would be more trivial to strip out the dag_id, task_id bits